test/unit_convex_hull.jl

```diff
@@ -6,7 +6,7 @@ for N in [Float64, Rational{Int}]
     # corner cases in dimension 1
     @test convex_hull([Vector{N}(undef, 0)]) == [Vector{N}(undef, 0)]
     @test convex_hull([[N(0)]]) == [[N(0)]]
-    @test convex_hull([[N(2)], [N(1)]]) == [[N(1)], [N(2)]]
+    @test convex_hull([[N(2)], [N(1)]]) == [[N(2)], [N(1)]]
```

Comment:
```suggestion
    @test ispermutation(convex_hull([N[2], N[1]]), [N[2], N[1]])
```
Or do you want to require that the implementation does not change the order?